storage/mysql/storage_test.go

```diff
@@ -198,45 +198,39 @@ func createLogNodesForTreeAtSize(t *testing.T, ts, rev int64) ([]stree.Node, err
 	// Unroll the map, which has deduped the updates for us and retained the latest
 	nodes := make([]stree.Node, 0, len(nodeMap))
 	for id, hash := range nodeMap {
-		nID, err := stree.NewNodeIDForTreeCoords(int64(id.Level), int64(id.Index), 64)
-		if err != nil {
-			t.Fatalf("failed to create NodeID for %+v: %v", id, err)
-		}
-		node := stree.Node{NodeID: nID, Hash: hash, NodeRevision: rev}
-		nodes = append(nodes, node)
+		nodes = append(nodes, stree.Node{ID: id, Hash: hash})
 	}
-
 	return nodes, nil
 }
 
 // TODO(pavelkalinnikov): Allow nodes to be out of order.
-func nodesAreEqual(lhs []stree.Node, rhs []stree.Node) error {
+func nodesAreEqual(lhs, rhs []stree.Node) error {
```

Comment:
Can we replace this with some variant of cmp.Diff? That did't exist when this was written + same for func below.

Reply:
The plan is to significantly refactor/simplify `SubtreeCache` anyway, so I will likely revisit this in follow-up PRs.